terasoluna-gfw-parent/pom.xml

```diff
@@ -413,7 +413,7 @@
     <cargo.daemon.url>http://localhost:18000</cargo.daemon.url>
     <cargo.container.id>tomcat8x</cargo.container.id>
     <cargo.container.url>http://archive.apache.org/dist/tomcat/tomcat-8/v${cargo.tomcat8.version}/bin/apache-tomcat-${cargo.tomcat8.version}.zip</cargo.container.url>
-    <cargo.tomcat8.version>8.5.39</cargo.tomcat8.version>
+    <cargo.tomcat8.version>8.5.38</cargo.tomcat8.version>
```

Comment:
- [ ] I recommend to unify the version of tomcat used by cargo with the version of the tomcat libraries used by the application.

Refer this. https://github.com/terasolunaorg/terasoluna-gfw/blob/bd5ac55536b992d27b9bbc657400b3389ffe0dac/terasoluna-gfw-parent/pom.xml#L505-L556

Reply:
80ca96fdf7d9a3f1628af8910b1b1dcd874ae364

I fixed in above commit .